### PR TITLE
[ELI5] Tweaks to hosting provider

### PR DESCRIPTION
**Pull request by mozzius**
Screenshot 2024-12-03 at 23 23 47

Also, if you're not logged into any other accounts, you get a new user tip in the service provider dialog

Screenshot 2024-12-03 at 23 21 53

### CODE REVIEW

**Comment by emilyliu7321** [comment]
Some copy nits in this dialogue. Also to confirm, this dialogue only pops up if you're a new user _and_ you click into the ✍️ icon right?
image

`Choose Service. Select the service that hosts your data.` --> `Choose your account provider.` (no second line needed)

`Bluesky | Custom` --> `Bluesky Social | Custom`

🌱  tooltip: `Bluesky is an open network where you can choose your own provider. If you're new here, we recommend sticking with the default Bluesky Social option.`

bottom paragraph: `Developers can host their own servers. Learn more.`

**Review by estrattonbailey — APPROVED** [review]
